### ISSUE TITLE
On pressing the 'Join event' button, there is no toast message.

### ISSUE THREAD

**Issue by poorboy23** [rolanorebelo/ElderApp#14]
When the user presses the 'Join Event' button, there should be a toast message indicating success or failure of the operation to let the user know the outcome. This ensures proper accessibility in the app.

Actual Behavior:
When the user presses the 'Join Event' button, there's no toast message indicating the result.

Expected Behavior:
When the user presses the 'Join Event' button, there's a toast message indicating the result.

Before Fix:

https://github.com/rolanorebelo/ElderApp/assets/61705777/ffce9447-9bfe-4436-ae58-4e794aac4910

Actual Fix:

https://github.com/rolanorebelo/ElderApp/assets/61705777/7c628b36-ceb4-4ba5-b58a-1d559ff25543

**Comment by poorboy23** [comment]
This issue was fixed by adding a Toast message to the addDoc function. So, when the user press join event, the data is added to the database and the toast message is triggered at the same time on success of database entry.

`
    // Define the event data to store in Firestore
    const eventData = {
      eventId: event.id,       // Event ID
      userId: user.uid,       // User ID
      joinDateTime: new Date().toISOString(),  // Current date and time in ISO format
    };

    // Reference to the Firestore collection "event_joins" (you can change the collection name)
    const eventJoinsRef = collection(getFirestore(), "event_joins");

    // Add a new document to the "event_joins" collection
    addDoc(eventJoinsRef, eventData)
      .then(() => {
        console.log("User joined the event successfully!");
        Toast.show({
          type: 'success',
          position: 'bottom',
          text1: 'User joined the event successfully!',
          visibilityTime: 3000, // Adjust as needed
        });
      })
      .catch((error) => {
        console.error("Error joining the event: ", error);
      });
  };`
